### ISSUE TITLE
Bug fix release

### ISSUE THREAD

**Issue by znicholls** [IAMconsortium/pyam#259]
@danielhuppmann or @gidden any chance of a hotfix release? At the moment pip installing doesn't work because you get pandas 0.25 which is really annoying

cc @lewisjared

**Comment by jkikstra** [comment]
@gidden @danielhuppmann I've also just bumped into this issue when installing `ixmp` and `message_ix` from source code, then manually doing 'pip install pyam-iamc' and `pip install .[tests,reporting]` and then running `py.test tests`:
```
=============================================== test session starts ================================================
platform win32 -- Python 3.7.3, pytest-5.2.0, py-1.8.0, pluggy-0.13.0
message_ix location: C:\Users\kikstra\AppData\Local\Continuum\anaconda3\lib\site-packages\message_ix
ixmp config: {'DB_CONFIG_PATH': None, 'DEFAULT_DBPROPS_FILE': None, 'DEFAULT_LOCAL_DB_PATH': WindowsPath('C:/Users/kikstra/.local/share/ixmp/localdb/default')}```

**Comment by byersiiasa** [comment]
Same occurs with the conda-forge install. Seems that only git master avoids this error

**Comment by khaeru** [comment]
We have new users such as @jkikstra trying to use the message_ix reporting features to convert to pyam. Because #246 is still unreleased, I have to inform them to work around by installing `master`.

**Comment by Jihoon** [comment]
I am also having this issue.

**Comment by gidden** [comment]
Hi all - with #277 I think we are ready to do a release (also to help backups with silicone, openscm, etc.). @danielhuppmann can you take us across the finish line here?

**Comment by danielhuppmann** [comment]
resolved with release 0.3.0, https://github.com/IAMconsortium/pyam/releases/tag/v0.3.0
